bvm/ballerina-core/src/main/java/org/ballerinalang/model/util/XMLUtils.java

```diff
@@ -655,4 +657,28 @@ private static QName getQName(String localName, String namespaceUri, String pref
         }
         return qname;
     }
+
+    /**
+     * Create an OMElement from an XML fragment given as a string.
+     *
+     * @param xmlFragment the well-formed XML fragment
+     * @return The OMElement created out of the string XML fragment.
+     * @throws XMLStreamException
+     */
+    public static OMElement stringToOM(String xmlFragment) throws XMLStreamException {
+        return stringToOM(OMAbstractFactory.getOMFactory(), xmlFragment);
+    }
+
+    /**
+     * Create an OMElement from an XML fragment given as a string.
+     *
+     * @param omFactory the factory used to build the object model
+     * @param xmlFragment the well-formed XML fragment
+     * @return The OMElement created out of the string XML fragment.
+     * @throws XMLStreamException
```

Comment:
Shall we add description for exception

Reply:
This is from AXIOMUtil [1, which we used before this change (since the AXIOMUtil is not flexible enough, it had to be moved.)  We'll have to dig into AXIOM to get exception details. Will look into that. I'll check and try to improve that. 

[1] https://ws.apache.org/axiom/code-coverage/org.apache.axiom.om.util/AXIOMUtil.java.html

Reply:
Done